src/main/resources/static/api.yml

```diff
@@ -1866,6 +1866,46 @@ paths:
         500:
           $ref: '_shared.yml#/components/responses/500Response'
       x-codegen-request-body-name: body
+  /applications/{applicationId}/timeline:
+    get:
+      tags:
+        - Application data timeline
+      summary: Returns domain event summary
+      parameters:
+        - name: applicationId
+          in: path
+          description: ID of the application
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: X-Service-Name
+          in: header
+          required: true
+          description: If given, only users for this service will be returned
+          schema:
+            $ref: '_shared.yml#/components/schemas/ServiceName'
+      responses:
+        200:
+          description: successful operation
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '_shared.yml#/components/schemas/DomainEventSummary'
```

Comment:
I'd feel a bit happier if we had a new object for this - something like `TimelineEvent`, that way if we want to change any implementation of the `DomainEventSummary`, then we can keep the `TimelineEvent` the same (or vice versa)